common/src/attestations/dispute_manager.rs

```diff
@@ -7,8 +7,6 @@ use std::time::Duration;
 use thegraph_core::Address;
 use tokio::sync::watch::{self, Receiver};
 use tokio::time::{self, sleep};
-use alloy::primitives::Address;
```

Comment:
Could you use `use alloy::primitives::Address;` instead of `use thegraph_core::Address;`

Reply:
oh , sorry